app/actions/definitions/teams.tsx

```diff
@@ -1,40 +1,64 @@
+import { PlusIcon } from "outline-icons";
 import * as React from "react";
 import styled from "styled-components";
+import TeamNew from "~/scenes/TeamNew";
 import { createAction } from "~/actions";
 import { loadSessionsFromCookie } from "~/hooks/useSessions";
 
-export const changeTeam = createAction({
-  name: ({ t }) => t("Switch team"),
+export const switchTeamList = getSessions().map((session) => {
+  return createAction({
+    name: session.name,
+    section: "Switch Team",
+    keywords: "change switch workspace organization team",
+    icon: () => <Logo alt={session.name} src={session.logoUrl} />,
+    visible: ({ currentTeamId }) => currentTeamId !== session.teamId,
+    perform: () => (window.location.href = session.url),
+  });
+});
+
+const switchTeam = createAction({
+  name: ({ t, isContextMenu }) =>
+    isContextMenu ? t("Teams") : t("Switch team"),
   placeholder: ({ t }) => t("Select a team"),
-  keywords: "change workspace organization",
-  section: "Account",
-  visible: ({ currentTeamId }) => {
-    const sessions = loadSessionsFromCookie();
-    const otherSessions = sessions.filter(
-      (session) => session.teamId !== currentTeamId
-    );
-    return otherSessions.length > 0;
-  },
-  children: ({ currentTeamId }) => {
-    const sessions = loadSessionsFromCookie();
-    const otherSessions = sessions.filter(
-      (session) => session.teamId !== currentTeamId
-    );
+  keywords: "change switch workspace organization team",
+  section: "Team",
+  visible: ({ currentTeamId }) =>
+    getSessions({ exclude: currentTeamId }).length > 0,
+  children: switchTeamList,
+});
 
-    return otherSessions.map((session) => ({
-      id: session.url,
-      name: session.name,
-      section: "Account",
-      icon: <Logo alt={session.name} src={session.logoUrl} />,
-      perform: () => (window.location.href = session.url),
-    }));
+export const createTeam = createAction({
+  name: ({ t }) => `${t("New team")}…`,
+  keywords: "create change switch workspace organization team",
+  section: "New Team",
```

Comment:
I think this is also fine under the same `Workspaces` section, it doesn't need to match the name of the action itself